sdk/eventhub/Microsoft.Azure.EventHubs/tests/Amqp/AmqpMEssageCoverterTests.cs

```diff
@@ -55,7 +55,8 @@ private static ArraySegment<byte>[] ReadMessagePayLoad(AmqpMessage message, int
                 ArraySegment<byte>[] messageBuffers = message.GetPayload(payloadSize, out more);
                 if (messageBuffers != null)
                 {
-                    foreach (var segment in messageBuffers) { message.CompletePayload(segment.Count); }
+                    foreach (var segment in messageBuffers)
+                    { message.CompletePayload(segment.Count); }
```

Comment:
Would it be more .NET to put it on separate lines?
```cs
{
    message.CompletePayload(segment.Count);
}
```

Reply:
Without a doubt;  however, this is track one code, so I'm making just the minimal set of changes.  :)